examples/follicle_viewer.py

```diff
@@ -18,23 +19,31 @@
 from qtpy.QtWidgets import QLabel, QVBoxLayout, QWidget
 from scipy import ndimage as ndi
 
-file_name = "A01_z1_links_170423_ovary.h5"
```

Comment:
I think it's fine for now, but it would be nice to have the option to select a specific file to load. Perhaps we can make a simple widget to load a particular file in a follow-up PR.